utils/utilityApi.js

```diff
@@ -133,4 +133,4 @@ const getLatestPGEBill = async meterId => {
   };
 };
 
-export default getLatestPGEBill;
+export {getLatestPGEBill};
```

Comment:
Destructured import/export to enable Sinon stubbing

Reply:
spaces again